test/test_scripts/test_shard_bam.py

```diff
@@ -0,0 +1,12 @@
+def test_shard_bam(script_runner):
```

Comment:
nice to have a test!

on the other hand, I do think testing based on stdout is ani-pattern.

Reply:
I think this is a bigger problem and share something in common with cromwell 2, so we should talk about it.

Reply:
Test now checks the contents of the output file much more rigorously.